tests/test_serialization.py

```diff
@@ -79,6 +81,45 @@ def test_save_load__from_filename(
         save_load_from_filename(cfg, resolve, expected, file_class)
 
 
+@dataclass
+class Dataclass:
+    number: int = 2
+    done: bool = True
+
+
+@attr.s(auto_attribs=True)
+class Attr:
+    number: int = 2
+    done: bool = True
+
+
+@pytest.mark.parametrize(
+    "input_,resolve,expected,file_class",
+    [
+        (Dataclass, False, {"number": 2, "done": True}, str),
+        (Attr, False, {"number": 2, "done": True}, str),
```

Comment:
for completeness, test instance as well.(Dataclass()).
Also, use better names. Dataclas and Attr are terrible :)